jwql/website/apps/jwql/monitor_pages/badpixel_monitor.py

```diff
@@ -0,0 +1,44 @@
+import os
```

Comment:
@bhilbert4 Is it ok to remove this file completely? Or should we leave it around until we are completely finished with this monitor?

Reply:
This file can be deleted. Was a false start a while ago.